wrangler/src/txFunctions/runCB.js

```diff
@@ -0,0 +1,197 @@
+import { response, Stopwatch } from 'cfw-easy-utils'
+import { Transaction, Networks, Keypair } from 'stellar-base'
+import BigNumber from 'bignumber.js'
+import moment from 'moment'
+import { find as loFind } from 'lodash'
+
+import txSponsorsSettle from '../txSponsors/settleCB'
+
+// TODO:
+  // Remove all old txSponsor stuff
+
+export default async ({ event, request, params }) => {
+  const { txFunctionHash } = params
+
+  const { value, metadata } = await TX_FUNCTIONS.getWithMetadata(txFunctionHash, 'arrayBuffer')
+
+  if (!value)
+    throw {status: 404, message: `txFunction could not be found this turret`}
+
+  const { length, txFunctionSignerPublicKey, txFunctionSignerSecret } = metadata
+
+  const txFunctionBuffer = Buffer.from(value)
+  const txFunction = txFunctionBuffer.slice(0, length).toString()
+
+  const body = await request.json()
+  const claimableBalanceToken = request.headers.get('authorization')?.split(' ')?.[1]
```

Comment:
Should we consider an expiring token? I know there are some concerns with token sniping.

Reply:
It's kind of an expiring token already. Either you run out of balance or the server claims before the secondary claimant date passes. It's not baked into the token but it's more or less baked into the claimable balance itself. Which is preferable imo. The token is the vehicle for the "actual token". Seems like potato potato but I'm happy to be proven wrong.